<launch>
  <arg name="rate" default="10" />
  <arg name="namespace" default="husky" />

  <group ns="$(arg namespace)">
    <node name="global_planner" pkg="larics_motion_planning" type="global_planner_node" output="screen">
      <param name="rate" value="$(arg rate)" />
      <param name="global_planner_config_file" value="$(find larics_motion_planning)/config/husky_config.yaml" />

      <param name="state_scale" value="0.05" />
      <param name="waypoints_scale" value="0.5" />

      <remap from="/husky/multi_dof_trajectory" to="/husky/trajectory" />
    </node>
  </group>
</launch>